build/upload_tarballs.jl

```diff
@@ -92,15 +64,9 @@ if abspath(PROGRAM_FILE) == @__FILE__
     # check that contents are tarballs with correct filename
     all(t -> !isnothing(match(TARBALL_REGEX, t)), readdir(TARBALL_DIR))
 
-    pkg_http_url = parse_git_remote_url(PKG_URL)
-    artifact_url = "$(pkg_http_url)/releases/download/$TAG"
+    artifact_url = gen_artifact_url(; repo_url=REPO_HTTPS_URL, tag=TAG, filename="")
     branch = LibGit2.with(LibGit2.branch, LibGit2.GitRepo(REPO_PATH))
 
     @info "Uploading tarballs to $artifact_url"
-    try
-        upload_to_github_release(TARBALL_DIR, artifact_url, branch)
-    catch e
-        # ghr() already prints an error message with diagnosis
-        @debug "Caught exception $(sprint(showerror, e, catch_backtrace()))"
-    end
```

Comment:
The `try`/`catch` here not only suppressed `ghr` errors but also the `throw`n errors from the Julia code